### PR TITLE
moving webdriver before ide, adding padding to table cells

### PR DESCRIPTION
**Pull request by ddavison**
also adding header over the respective kinds

screen shot 2015-03-20 at 12 07 54

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -17,23 +22,13 @@
...
-  <li>create scripts to aid in automation-aided exploratory testing</li>
-</ul>
-<br />
-<p>Then you want to use <a href="/projects/ide/">Selenium IDE</a>; a Firefox add-on that will do simple record-and-playback of interactions with the browser.</p>
-        </td>
-        <td><center><a href="/projects/webdriver/"><img alt="Selenium Logo" class="icon" src="images/selenium-logo.png" /></a><a href="/projects/webdriver/"><img alt="Selenium Grid Logo" class="icon" src="images/selenium-grid-logo.png" /></a></center><br />
-If you want to
-<ul>
-  <li>create robust, browser-based regression automation</li>
+  <li>create robust, browser-based regression automation suites and tests</li>
```
Comment (ddavison) [inline src/main/webapp/index.jsp]: also adding "suites and tests"

File: src/main/webapp/index.jsp [hunk 2]
```diff
@@ -42,6 +37,15 @@ If you want to
                 now also includes built-in grid capabilities.</p>
 
         </td>
+        <td><center><h3>Selenium IDE</h4><a href="/projects/ide/"><img alt="Selenium IDE Logo" class="icon" src="images/selenium-ide-logo.png" /></a></center><br />
```
Comment (andreastt) [inline src/main/webapp/index.jsp]: Don't use `<center>` or <br />` for styling.

**Comment by andreastt** [comment]
LGTM.  Can you fixup the nitpick about styling?

File: src/main/webapp/index.jsp (hunk 2 above)
Comment (lukeis) [inline src/main/webapp/index.jsp]: he simply moved the line.... this was here long ago

Comment (andreastt) [inline src/main/webapp/index.jsp]: OK, let's fix this later then.

**Pull request merged**